### PR TITLE
add withdraw instruction for vote program

### PR DESCRIPTION
**Pull request by conorpp**
Adds the withdraw instruction for vote program:
https://docs.rs/solana-vote-program/latest/solana_vote_program/vote_instruction/enum.VoteInstruction.html#variant.Withdraw

### CODE REVIEW

File: programs/vote/Withdraw.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+	parent.Child(format.Program(ProgramName, ProgramID)).
+		//
+		ParentFunc(func(programBranch treeout.Branches) {
+			programBranch.Child(format.Instruction("Withdraw")).
+				//
+				ParentFunc(func(instructionBranch treeout.Branches) {
+
+					// Parameters of the instruction:
+					instructionBranch.Child("Params").ParentFunc(func(paramsBranch treeout.Branches) {
+						paramsBranch.Child(format.Param("Lamports", *inst.Lamports))
```
Comment (gagliardetto) [inline programs/vote/Withdraw.go:117]:
```suggestion
						paramsBranch.Child(format.Param("Lamports", inst.Lamports))
```

will avoid panic

File: programs/vote/Withdraw.go [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+					})
+
+					// Accounts of the instruction:
+					instructionBranch.Child("Accounts").ParentFunc(func(accountsBranch treeout.Branches) {
+						accountsBranch.Child(format.Meta("                Vote", inst.AccountMetaSlice[0]))
```
Comment (gagliardetto) [inline programs/vote/Withdraw.go:122]:
```suggestion
						accountsBranch.Child(format.Meta("                Vote", inst.AccountMetaSlice.Get(0)))
```

File: programs/vote/Withdraw.go [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+						accountsBranch.Child(format.Meta("           Recipient", inst.AccountMetaSlice[1]))
```
Comment (gagliardetto) [inline programs/vote/Withdraw.go:123]:
```suggestion
						accountsBranch.Child(format.Meta("           Recipient", inst.AccountMetaSlice.Get(1)))
```

File: programs/vote/Withdraw.go [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+						accountsBranch.Child(format.Meta("AuthorizedWithdrawer", inst.AccountMetaSlice[2]))
```
Comment (gagliardetto) [inline programs/vote/Withdraw.go:124]:
```suggestion
						accountsBranch.Child(format.Meta("AuthorizedWithdrawer", inst.AccountMetaSlice.Get(2)))
```

**Review by gagliardetto — CHANGES_REQUESTED** [review]

**Comment by conorpp** [comment]
fixed!

**Pull request merged**